spec/services/search/radius_builder_spec.rb

```diff
@@ -35,6 +35,14 @@
           end
         end
       end
+
+      context "when radius is not an integer" do
+        let(:radius) { "; DROP TABLE Vacancies; " }
```

Comment:
Asserting that user input is sanitised by the Search::RadiusBuilder class to avoid sql injection